manifest.json

```diff
@@ -39,7 +39,8 @@
                 "scripts/highlighting.js",
                 "scripts/repoHandlers.js",
                 "scripts/orchestrator.js",
-                "scripts/init.js"
+                "scripts/init.js",
+                "scripts/signedOutIndicator.js"
```

Comment:
I think you will have to add this script on the top (at least before `scripts/orchestrator.js` file for it to be importable. 

Let me know if what you have created right now is working without issues.

Reply:
This works without issues as of now, can change if required if there is some ordering preference.